Documentation/ContentMapping/Index.rst

```diff
@@ -1,186 +1,372 @@
 
 .. highlight:: typoscript
 
-Create a new directory :file:`Configuration/TypoScript/Helper/` and inside this
-directory, a new file called :file:`DynamicContent.typoscript` with the following
+Create a new directory :file:`Configuration/TsConfig/Page/` and inside this
+directory, a new file called :file:`Page.tsconfig` with the following
 content::
 
-   lib.dynamicContent = COA
-   lib.dynamicContent {
-      10 = LOAD_REGISTER
-      10 {
-         colPos.cObject = TEXT
-         colPos.cObject {
-            field = colPos
-            ifEmpty.cObject = TEXT
-            ifEmpty.cObject {
-               value.current = 1
-               ifEmpty = 0
-            }
-         }
-         pageUid.cObject = TEXT
-         pageUid.cObject {
-            field = pageUid
-            ifEmpty.data = TSFE:id
-         }
-         contentFromPid.cObject = TEXT
-         contentFromPid.cObject {
-            data = DB:pages:{register:pageUid}:content_from_pid
-            data.insertData = 1
-         }
-         wrap.cObject = TEXT
-         wrap.cObject {
-            field = wrap
-         }
-      }
-      20 = CONTENT
-      20 {
-         table = tt_content
-         select {
-            includeRecordsWithoutDefaultTranslation = 1
-            orderBy = sorting
-            where = {#colPos}={register:colPos}
-            where.insertData = 1
-            pidInList.data = register:pageUid
-            pidInList.override.data = register:contentFromPid
-         }
-         stdWrap {
-            dataWrap = {register:wrap}
-            required = 1
-         }
-      }
-      30 = RESTORE_REGISTER
-    }
+   @import 'EXT:site_package/Configuration/TsConfig/Page/PageLayout/*.tsconfig'
+
+This file imports all files ending on `.tsconfig` from the specified folder.
+The file :file:`Page.tsconfig` could for example handle other page TsConfig
+configurations or their imports.
+
+Then create a file
+:file:`Configuration/TsConfig/Page/PageLayout/Default.tsconfig` with the
+following content::
+
+   mod.web_layout.BackendLayouts {
+       Default {
+           title = Default Layout
+           config {
+               backend_layout {
+                   colCount = 1
+                   rowCount = 2
+                   rows {
+                       1 {
+                           columns {
+                               1 {
+                                   name = Jumbotron
+                                   colPos = 1
+                               }
+                           }
+                       }
+                       2 {
+                           columns {
+                               1 {
+                                   name = Main Content
+                                   colPos = 0
+                               }
+                           }
+                       }
+                   }
+               }
+           }
+       }
+   }
+
+After clearing the all caches the new backend layout is available in the page
```

Comment:
"the cache" - sorry, I have no deeper knowledge which and if a cache needs to be cleared for PageTS

Reply:
they do, these files get cached... I added an explainaition